### ISSUE TITLE
Tags with Question Marks ("?") doesn't seem to download.

### ISSUE THREAD

**Issue by gabrielwoj** [woodenphone/Derpibooru-dl#15]
Hello, I've ran derpibooru-dl today to get some pictures and I've noticed that one picture in specific wasn't downloaded (when I was browsing through the website). The picture was #955059 and these were the tags that I've set-up on the tag_list.txt:

+fluttershy, -explicit, -grimdark, -grotesque, id_number.gte:953913

I decided to try testing the "hugs?" tag alone, and it seems that the question mark is ignored when doing so, downloading only the "hugs" tag.

So the issue is, if you try using a tag which contains a question mark, it's not going to work, the same applies if the image you've got with your tag (Ex: +fluttershy) has a tag with question mark (which causes the image simply to not download).

This is the first time I'm using Github, so I'm sorry if I'm doing it wrong.
I'm using the .exe program on Windows 10 64bit.

**Comment by Nuke928** [comment]
I think it's because the ? is not escaped properly in the URL. Might try to fix it later.

**Comment by gabrielwoj** [comment]
Yes, the question mark gets switched to "%3F" in the URL, since it's a special character.

Thanks.

**Comment by Nuke928** [comment]
Are you using this release? https://github.com/woodenphone/Derpibooru-dl/releases/tag/derpibooru_dl-2014-11-4

The program has changed a bit and I can't seem to get it working right now, neither did the documentation mention of an additional library (derpybooru). @woodenphone should handle this.

**Comment by gabrielwoj** [comment]
Yup, I'm using the 2014-11-4 release. I forgot to mention that I've put the API key on my settings file just fine, so the issue is not with the API key.
